### PR TITLE
Z-Ordering

### PR DESCRIPTION
**Pull request by bagnell**
Adds `Scene.groundPrimitives`. Use the `raise`/`lower` functions for correct z-ordering.

### CODE REVIEW

File: Source/Scene/Scene.js [hunk 1]
```diff
@@ -1606,6 +1620,7 @@ define([
         }
 
         scene._primitives.update(context, frameState, commandList);
+        scene._groundPrimitives.update(context, frameState, commandList);
```
Comment (pjcozzi) [inline Source/Scene/Scene.js]: Do you think we should do this before `_primitives.update` so all ground primitives are drawn below all primitives if both were to have `GroundPrimitive`s?

**Comment by pjcozzi** [comment]
Code looks good, just that question.

**Comment by bagnell** [comment]
@pjcozzi This is ready,

**Comment by pjcozzi** [comment]
The amazement continues.

**Pull request merged**